prefect-worker/plural/recipes/prefect-worker-aws.yaml

```diff
@@ -19,6 +19,9 @@ sections:
   - name: workspaceId
     documentation: the workspace id of your Prefect workspace
     type: STRING
+  - name: workPool
```

Comment:
Wouldn't you want to template the value you get from this recipe input into the helm chart using the `yalues.yaml.tpl` file?

Reply:
hmm - I think it is [here](https://github.com/pluralsh/plural-artifacts/blob/main/prefect-worker/helm/prefect-worker/values.yaml.tpl#L3) but lemme know if there is something more I'm missing!  Was more or less following what had been done previously for the accountId and workspaceId inputs.